core/backend/src/IModelDb.ts

```diff
@@ -799,6 +801,31 @@ export abstract class IModelDb extends IModel {
     return this.nativeDb.restartTxnSession();
   }
 
+  /** @internal */
+  public setSyncDataStore(store: IModelSyncDataStore.CloudAccess) {
+    this._syncDataStore = store;
+  }
+
+  /** @internal */
+  public getSyncDataStore() {
+    return this._syncDataStore;
+  }
+
+  /** @internal */
+  public async initSharedSchemaChannel() {
+    const store = this._syncDataStore;
+    if (!store) {
+      throw new IModelError(DbResult.BE_SQLITE_ERROR, "Sync data store clould access is not set.");
+    }
+    await store.withLockedDb("init shared schema channel", async () => {
+      await this.nativeDb.saveChanges();
+      await this.nativeDb.sharedChannelInit(store.getUri());
```

Comment:
why is this called a "shared channel"? Is it a channel?

Reply:
The correct term was  "Shared Schema Channel". I will fix it.

Reply:
but is it a channel? Why?

Reply:
Fixed. All Shared/Channel -> Schema Sync